### DISCUSSION TITLE
tiktok coins and problem with youtube creator

### DISCUSSION THREAD

**Discussion by Jan204** [3dproger/AxelChat#556] [Ideas]
hey, is it possible for axel chat to put together the tiktok coin combos? If several combos come, you can't read anything anymore because it spams everything.
I also haven't managed to get YouTube to save the creator and I have to copy a new link every time :(

**Comment by 3dproger** [comment]
Hi,

> hey, is it possible for axel chat to put together the tiktok coin combos? If several combos come, you can't read anything anymore because it spams everything.

Ok, I figured out the problem, I'll try to fix it in one of the future updates. Just in case, I created a issue for this: https://github.com/3dproger/AxelChat/issues/558

**Comment by 3dproger** [comment]
> I also haven't managed to get YouTube to save the creator and I have to copy a new link every time :(

Can you tell me what link you are trying to insert into the AxelChat? If you don't want to specify a link to your channel, you can specify a link to another channel, as long as it gives me an idea of ​​what the link you were trying to insert looks like. Perhaps I haven't implemented the type of links you are using yet

**Comment by 3dproger** [comment]
I probably found your channel. I see that there is an error, but this is not because the channel is not found, but because the stream is not running, or this stream is not public, or the chat on the stream is disabled, or the stream has age restrictions
![изображение](https://github.com/user-attachments/assets/326ab220-8e08-41d8-a055-81c9d8d37592)

**Reply by 3dproger** [reply]
I also specified the channel like this: `@Jan204`. But please note that you cannot specify a channel in this form: `Jan204` that is, without the symbol `@`. I haven't written about this anywhere yet, but I'll try to add information about it to the YouTube connection settings

**Comment by Jan204** [comment]
Hey,

thanks for the quick reply. unfortunately i didn't get a push message.
I enter it like this, for example:

![Screenshot 2024-10-03 174657](https://github.com/user-attachments/assets/fecb6b60-a04e-4389-b791-bdd8f7aaddc6)

or even if I only use @username, it does not work

**Reply by 3dproger** [reply]
To be honest, I can't reproduce the problem yet
![изображение](https://github.com/user-attachments/assets/daeb5901-7a3a-4615-aa4a-8d7477e26f45)

**Reply by 3dproger** [reply]
Can you send me the Logs? Either in text or just a screenshot, like below:
![изображение](https://github.com/user-attachments/assets/e6ff4879-66f6-4da7-9b4d-bc4d38226e1d)

**Reply by 3dproger** [reply]
You can open the logs here:
![изображение](https://github.com/user-attachments/assets/7fcdfd58-c00d-4cef-95ee-1b0cb73b0964)

**Comment by 3dproger** [comment]
I think I just encountered this problem too. It's probably related to YouTube instability. I'll try to fix this issue in the next update

**Comment by Jan204** [comment]
After Update:
[AxelChat-log-2024-10-08T13-33-58.487.txt](https://github.com/user-attachments/files/17292559/AxelChat-log-2024-10-08T13-33-58.487.txt)

sorry, i look now every day in the forum

**Reply by 3dproger** [reply]
No problem, thanks for sending the logs. I'll try to fix it

**Reply by Jan204** [reply]
thank u :)

**Comment by Jan204** [comment]
hey,
quick feedback:
unfortunately still present in beta 13.

**Comment by Jan204** [comment]
![Screenshot 2024-12-11 170634](https://github.com/user-attachments/assets/8bffcf4f-720f-4380-a94d-ee171a3a76aa)
so only slightly better, but still slightly faulty.

**Comment by 3dproger** [comment]
Hi! I made an update 0.38.0 in which I tried to fix the work with YouTube. I hope it will work well now

**Comment by 3dproger** [comment]
I think the discussion can be closed. If there are questions again, you can reopen the discussion or create a new one
